test/app.spec.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env mocha */
+const expect = require('chai').expect
+const request = require('request')
+const cheerio = require('cheerio')
+
+const app = require('../app.js')
+const lib = require('../lib/helper-functions')
+
+const serverOptions = {
```

Comment:
I'm unclear what this is doing – is it options used to create a server used for the test, or parameters used when making a request?

Reply:
parameters used when making a request. I should probably make that more obvious through naming